### PR TITLE
chore: patch bulk query tool

### PR DESCRIPTION
**Pull request by Goncharo**
refs. metriport/metriport-internal#1040

### Dependencies

n/a

### Description

patch bulk query tool, add whitelist, only kick off query, don't wait for completion

### Release Plan

asap

### CODE REVIEW

File: packages/utils/src/bulk-query-patients.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import * as dotenv from "dotenv";
 dotenv.config();
 // keep that ^ on top
-import { ConsolidatedCountResponse, MetriportMedicalApi, PatientDTO } from "@metriport/api-sdk";
+import { ConsolidatedCountResponse, MetriportMedicalApi, Patient } from "@metriport/api-sdk";
```
Comment (RamilGaripov) [inline packages/utils/src/bulk-query-patients.ts:8]: Why was this reverted? listPatients returns PatientDTO objects

**Review by RamilGaripov — APPROVED** [review]
Approved with a comment.

File: packages/utils/src/bulk-query-patients.ts [hunk 2]
```diff
@@ -44,7 +46,7 @@ function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-async function queryDocsForPatient(patient: PatientDTO) {
+async function queryDocsForPatient(patient: Patient) {
```
Comment (leite08) [inline packages/utils/src/bulk-query-patients.ts:49]: ditto PatientDTO

File: packages/utils/src/bulk-query-patients.ts [hunk 3]
```diff
@@ -63,14 +65,11 @@ async function queryDocsForPatient(patient: PatientDTO) {
...
+      // ensure at least 1 doc was returned
       if (
         docQueryStatus.download &&
-        docQueryStatus.download.status === "completed" &&
-        docQueryStatus.download.successful &&
-        docQueryStatus.download.successful > 0 &&
-        (!docQueryStatus.convert ||
-          (docQueryStatus.convert && docQueryStatus.convert.status === "completed"))
+        docQueryStatus.download.total &&
+        docQueryStatus.download.total > 0
```
Comment (leite08) [inline packages/utils/src/bulk-query-patients.ts:72]: What's the reason for changing this logic?

Comment (Goncharo) [reply packages/utils/src/bulk-query-patients.ts:72]: Intention here is to kick off queries, takes too many resources to track completion

**Review by leite08 — APPROVED** [review]

**Pull request merged**